<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QWidget" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QPushButton" name="pBtn_close">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>cerrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Address">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>IP: 127.0.0.1 , port: 1234</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Message">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Mensaje Enviado</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
